src/patternfly/demos/MasterDetail/examples/MasterDetail.md

```diff
@@ -0,0 +1,401 @@
+---
+title: Master Detail
+section: demos
+---
+
+import './MasterDetail.css'
+
+## Demos
+
+```hbs title=Master-detail-table-expanded isFullscreen
+{{#> master-detail-template
+  master-detail-template--id="master-detail-table-expanded-example"
+  master-detail-template--title="Master detail expanded, with table and .pf-m-inline modifier demo"
+  master-detail-template--description="Body text should be Red Hat Text Regular at 16px. It should have leading of 24px because of its relative line height of 1.5"
+  master-detail-template--modifier="pf-m-light"
+  }}
+
+  {{#> divider divider--type="div"}}{{/divider}}
+  {{#> page-main-section page-main-section--modifier="pf-m-no-padding"}}
+
+    <!-- Drawer -->
+    {{#> drawer
+      drawer--id=(concat master-detail-template--id '-drawer')
+      drawer-panel--IsOpen="true"
+      drawer--modifier="pf-m-inline-on-2xl"
```

Comment:
This modifier name makes me think the the component turns to inline at that breakpoint. Is this correct?

Reply:
Yes, that is correct. `panel` and `content` widths will respond to each other rather than the `panel` overlapping the `content` at the `-2xl` breakpoint (in this case).

Reply:
Ok does this relate to this? When inline, you would click an item in the list and then the content would appear on top?
<img width="671" alt="Screen Shot 2020-02-12 at 9 58 58 AM" src="https://user-images.githubusercontent.com/20118816/74346983-62fa2480-4d7e-11ea-88c1-944ac992329a.png">

Reply:
I wonder if you should show an example of this at that breakpoint where the content is expanded on top of the list items?